scrapers/Diabolic.yml

```diff
@@ -0,0 +1,47 @@
+name: Diabolic
+sceneByURL:
+  - action: scrapeXPath
+    scraper: sceneScraper
+    url:
+      - diabolic.com/en/video/
+movieByURL:
+  - action: scrapeXPath
+    scraper: movieScraper
+    url:
+      - diabolic.com/en/movie/
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h3[@class="sceneTitle"]
+      Date:
+        selector: //li[@class="updatedDate"]/text()
+        postProcess:
+          - replace:
+              - regex: ":"
+                with: ""
+          - parseDate: 2006-01-02
+      Performers:
+        Name: //div[@class="sceneCol sceneColActors"]/a
+      Tags:
+        Name: //div[@class="sceneCol sceneColCategories"]/a
```

Comment:
It seems that to get all tags we need to use CDP as tags that are displayed in the actual page are updated using a script.
(the main tags from the above are usually below 10)
```yaml
      Tags:
        Name: //div[contains(@class,"ScenePlayerHeaderDesktop-CategoriesContainer")]//a/@title
```
using the above and CDP seems to get all the visible tags

Reply:
Did not even notice that, should be done now :-)